### DISCUSSION TITLE
token重新获取多次还是不行，全部删了重新弄也不行。

### DISCUSSION THREAD

**Discussion by eightsheep67** [monlor/docker-xiaoya#95] [General]
xiaoya-alist-1     | ERRO[2024-06-11 02:33:47] failed get enabled storages: TokenVerifyFailed:The token was expected to have 3 parts, but got 1.
xiaoya-alist-1     | failed init storage
xiaoya-alist-1     | github.com/alist-org/alist/v3/internal/op.initStorage
xiaoya-alist-1     |    /app/internal/op/storage.go:99
xiaoya-alist-1     | github.com/alist-org/alist/v3/internal/op.LoadStorage
xiaoya-alist-1     |    /app/internal/op/storage.go:80
xiaoya-alist-1     | github.com/alist-org/alist/v3/internal/bootstrap.LoadStorages.func1
xiaoya-alist-1     |    /app/internal/bootstrap/storage.go:23
xiaoya-alist-1     | runtime.goexit
xiaoya-alist-1     |    /usr/lib/go/src/runtime/asm_amd64.s:1594

**Comment by xiaoweihong** [comment]
可能是被阿里云风控了。重启下路由器重新获取公网ip，或者过几天再试试
